### DISCUSSION TITLE
How can you make a piston platform portalable? (yes this is a question about BEEmod technically)

### DISCUSSION THREAD

**Discussion by ghost** [BEEmod/BEE2-items#4190] [Tech Support / Q&A] [answered]
I recently made an issue about BEEmod and I've got another one. When you right click the piston platform in BEE2 v4.36.1 it says if you change the 'cube type' you can make the surface portalable, but there is only 'delete' in portal 2 when you export. And yes, I know I'll need to run some console commands to make it work. Any help appreciated!

**Comment by vrad-exe** [comment, marked as answer]
You can definitely change the surface type that way, I'm not sure what would be wrong. ~But please don't use issues to ask questions, we have a Discord server.~
